### ISSUE TITLE
Activate Malware GuardDuty Feature

### ISSUE THREAD

**Issue by silavjy** [digitc1/AWSLandingZone#182]
Please enable the new GD module:

https://aws.amazon.com/about-aws/whats-new/2022/07/malware-protection-feature-amazon-guardduty/

For all EC AWS accounts.

**Comment by silavjy** [comment]
This feature is on hold, it's not possible to implement at this point due to the way our organisation is setup and the way guardduty administrator account is required to be setup (single account as opposed to the multiple seclog accounts we have).

Pending on update from AWS.

**Comment by silavjy** [comment]
addressed on the upcoming AWS LZ
